classify.c

```diff
@@ -280,6 +280,13 @@ static void add_banned_irq(int irq, GList **list)
 	return;
 }
 
+#ifdef AARCH64
+void add_banned_list_irq(int irq)
+{
+	add_banned_irq(irq, &banned_irqs);
+}
+#endif
+
```

Comment:
This still doesn't make any sense to me.  Why are you not just calling add_banned_irq directly?

Reply:
add_banned_irq is static in classify.c, so it cannot be called directly in procinterrupts.c

Reply:
so you wrote a new function that does the same thing?  why not just bifurcate add_banned_irq to __add_banned_irq(irq, list), and add_banned_irq(irq), and then make add_banned_irq non-static.  Theres no need to create another function with a vaguely simmilar name to do the same work

Reply:
this looks better, but this doesn't need to be an AARCH64 only option, it can be done for all arches, and so avoid the need for the ifdef

Reply:
this looks much better